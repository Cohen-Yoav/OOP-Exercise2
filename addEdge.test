# addEdge method Testing file

# check that the Edge was added to the graph
CreateNode n1 5
CreateNode n2 2
CreateGraph graph1
AddNode graph1 n1
AddNode graph1 n2
AddEdge graph1 n1 n2
AddEdge graph1 n1 n2
ListChildren graph1 n1
ListChildren graph1 n2

CreateNode n3 9
CreateNode n4 4
AddEdge graph1 n3 n2
AddEdge graph1 n1 n4
AddEdge graph1 n3 n4
AddEdge graph1 n1 n1
ListChildren graph1 n1

