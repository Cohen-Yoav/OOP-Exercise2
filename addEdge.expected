# addEdge method Testing file

# check that the Edge was added to the graph
created node n1 with cost 5
created node n2 with cost 2
created graph graph1
added node n1 to graph1
added node n2 to graph1
added edge from n1 to n2 in graph1
the children of n1 in graph1 are: n2
the children of n2 in graph1 are:

created node n3 with cost 9
created node n4 with cost 4
added edge from n1 to n1 in graph1
the children of n1 in graph1 are: n1 n2

